gtfs/spec/en/reference.md

```diff
@@ -269,9 +271,9 @@ File: **Optional**
 |  `price` | Non-negative float | **Required** | Fare price, in the unit specified by `currency_type`. |
 |  `currency_type` | Currency code | **Required** | Currency used to pay the fare. |
 |  `payment_method` | Enum | **Required** | Indicates when the fare must be paid. Valid options are:<br><br>`0` - Fare is paid on board.<br>`1` - Fare must be paid before boarding. |
-|  `transfers` | Enum | **Required** | Indicates the number of transfers permitted on this fare. The fact that this field can be left empty is an exception to the requirement that a Required field must not be empty. Valid options are:<br><br>`0` - No transfers permitted on this fare.<br>`1` - Riders may transfer once.<br>`2` - Riders may transfer twice.<br>empty - Unlimited transfers are permitted. |
+|  `transfers` | Enum | **Required** | Indicates the number of transfers permitted on this fare. Valid options are:<br><br>`0` - No transfers permitted on this fare.<br>`1` - Riders can transfer once.<br>`2` - Riders can transfer twice.<br>empty - Unlimited transfers are permitted. |
```

Comment:
Why is the word "can" substituted for "may" here? The change is the reverse of what's being done elsewhere in the document. These words are describing what the rider is allowed to do, not requirement levels in the specification so they are not really covered by RFC 2119. If the intent is to _avoid_ confusion with the meaning of "optional" given in RFC 2119, I suggest we use the words "is permitted" or "is allowed".

Reply:
Good point. Since this is explicit about "the rider", perhaps we can keep "may".